_includes/contribute-tutorial-lang.html

```diff
@@ -4,7 +4,7 @@
       <div class="column">
         <div class="snippet-wrapper">
           <div class="snippet has-text-centered cflt-tabs">
-            <div class="title">A tutorial for this language is available in <a href="https://docs.confluent.io/platform/current/tutorials/examples/clients/docs/clients-all-examples.html">Confluent documentation</a></div>
+            <div class="title">A tutorial for this language is available <a href="https://developer.confluent.io/get-started">here</a></div>
```

Comment:
I've had "here" links flagged a bunch in the past, FYC:
```suggestion
            <div class="title">A tutorial for this language is available on <a href="https://developer.confluent.io/get-started">Confluent Developer</a></div>
```

Reply:
I almost wrote `Confluent Developer` but KT will be part of CD very shortly, so it would refer to itself

Reply:
I still suggest some language besides "here".  Additionally it might be nice if we linked the user directly to the specific language page as with this link they land on a banner that says Java and that might be enough for them to back away

Reply:
Updated in latest commit